android/src/main/java/com/magicleap/magicscript/scene/nodes/layouts/UiLinearLayout.kt

```diff
@@ -74,31 +80,20 @@ class UiLinearLayout @JvmOverloads constructor(
         )
     }
 
-    private fun setOrientation(props: Bundle) {
-        if (props.containsKey(PROP_ORIENTATION)) {
-            val isVertical =
-                props.getString(PROP_ORIENTATION, DEFAULT_ORIENTATION) == ORIENTATION_VERTICAL
-            (layoutManager as LinearLayoutManager).isVertical = isVertical
-            requestLayout()
-        }
-    }
+    override fun getLayoutParams(): LinearLayoutParams {
+        val orientation = properties.getString(PROP_ORIENTATION, DEFAULT_ORIENTATION)
+        val itemPadding = properties.read(PROP_DEFAULT_ITEM_PADDING) ?: Padding()
+        val itemAlignment = properties.read<Alignment>(PROP_DEFAULT_ITEM_ALIGNMENT)!!
```

Comment:
why `!!`?

Reply:
We put a default alignment into `properties` inside the `init` block.  In case the default value changes, we don't have to remember to change it also her

Else it would be like this: